build.gradle

```diff
@@ -174,23 +181,32 @@ tasks.named('idea').configure {
 // Lets copy the distributions from build/install directory to /bin and /lib
 // directories to be aligned with C* distribution format
 tasks.register('copyDist', Copy) {
-    from "$buildDir/install/$applicationName"
     into "$projectDir"
+
```

Comment:
Please do NOT do this.  The application / distribution plugins come with the proper bin scripts.

Reply:
we've moved `conf/` to the top level dir , I think we want to avoid copying `./build/install/conf` into top level `conf/`. Other things we should avoid copying is `LICENSE.txt`  and `agents`. I guess the question here is what exactly do we need from the `copyDist` task ? I'm not sure what the need for `lib/` is here. Looking at `install` dir I have all of these:

```
$ tree ./build/install
./build/install
└── cassandra-sidecar
    ├── LICENSE.txt
    ├── agents
    │   ├── jolokia-core-1.6.0.jar
    │   ├── jolokia-jvm-1.6.0-agent.jar
    │   └── json-simple-1.1.1.jar
    ├── bin
    │   ├── cassandra-sidecar
    │   └── cassandra-sidecar.bat
    ├── conf
    │   ├── logback.xml
    │   └── sidecar.yaml
    └── lib
        ├── annotations-2.26.12.jar
        ├── ......
```

The main motivation is avoid overwriting the conf directory , since that's the source of truth for changes. If you overwrite it with an old snapshot, you'll lose your local changes if you had any

Reply:
I understand that.  The distTar / distZip tasks already create bin scripts and lib scripts, so what your doing can in the best case be redundent, and in the worst case, copy the wrong scripts over.  It's not needed.

Reply:
I think I've started by removing the task, then figured someone might benefit from having the bin, lib. But I think we don't need this task really